CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+### citus v10.0.3 (March 16, 2021) ###
+
+* Prevents infinite recursion for queries that involve `UNION ALL`
+  below `JOIN`
+
+* Fixes a crash in queries with a modifying `CTE` and a `SELECT`
+  without `FROM`
+
+* Fixes upgrade and downgrade paths for `citus_update_table_statistics`
+
+* Fixes a bug that causes `SELECT` queries to use 2PC unnecessarily
+
+* Fixes a bug that might cause self-deadlocks with
+  `CREATE INDEX` / `REINDEX CONCURRENTLY` commands
+
+* Adds `citus.max_cached_connection_lifetime` GUC to set maximum connection
+  lifetime
+
+* Adds `citus.remote_copy_flush_threshold` GUC that controls
+  per-shard memory usages by `COPY`
+
+* Adds new UDF citus_get_active_worker_nodes
```

Comment:
maybe something like:

Adds `citus_get_active_worker_nodes` UDF that replaces `master_get_active_worker_nodes`